packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js

```diff
@@ -228,6 +263,16 @@ QUnit.module('options', {
         assert.equal(this.tabWidgetInstance.option('selectedItem'), this.items[1], 'option <selectedItem> of nested tabs widget successfully changed');
     });
 
+    QUnit.test('orientation tabs option test', function(assert) {
```

Comment:
Let's use more informative caption for test `orientation option should be passed to tabs correctly` or something like this

Reply:
Nice point, I will replace this caption.

Reply:
[feccc1664117de61365ba2df824d600fa6ed06f6]